Lib/calendar.py

```diff
@@ -609,7 +609,7 @@ def formatmonthname(self, theyear, themonth, withyear=True):
             s = month_name[themonth]
             if withyear:
                 s = '%s %s' % (s, theyear)
-            return '<tr><th colspan="7" class="month">%s</th></tr>' % s
+            return '<tr><th colspan="7" class="%s">%s</th></tr>' % (self.cssclass_month_head, s)
```

Comment:
I don't know why the logic in base class is repeated and maybe I am missing something. How about the below ?

```python
    def formatmonthname(self, theyear, themonth, withyear=True):
        with different_locale(self.locale):
            return super().formatmonthname(theyear, themonth, withyear)
```